### DISCUSSION TITLE
May I join your team

### DISCUSSION THREAD

**Discussion by NRanjan-17** [ROM-builders/temporary#9993] [General]
https://cirrus-ci.com/github/ROM-builders/temporary/pull/8603
https://github.com/NRanjan-17/device_xiaomi_mido
https://github.com/NRanjan-17/vendor_xiaomi_mido
https://github.com/NRanjan-17/kernel_xiaomi_mido

Please Check now am I eligible

**Comment by Apon77** [comment]
> https://cirrus-ci.com/github/ROM-builders/temporary/pull/8603 https://github.com/NRanjan-17/device_xiaomi_mido https://github.com/NRanjan-17/vendor_xiaomi_mido https://github.com/NRanjan-17/kernel_xiaomi_mido
> 
> Please Check now am I eligible

give telegram id

**Reply by NRanjan-17** [reply]
> > https://cirrus-ci.com/github/ROM-builders/temporary/pull/8603 https://github.com/NRanjan-17/device_xiaomi_mido https://github.com/NRanjan-17/vendor_xiaomi_mido https://github.com/NRanjan-17/kernel_xiaomi_mido
> > Please Check now am I eligible
> 
> give telegram id

@HyperPower17
